docs/releases/Embeddinator-4000_0.2.md

```diff
@@ -0,0 +1,60 @@
+# Embeddinator-4000 0.2
+
+_work in progress_
+
+
+Requirements
+============
+
+* macOS 10.12 (Sierra) or later;
+* Xcode 8.3.2 or later;
+* Mono 5.0 or later;
+
+Optional
+--------
+
+* [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/);
+* [Xamarin.iOS 10.11.0.136 or later](https://jenkins.mono-project.com/view/Xamarin.MaciOS/job/xamarin-macios-builds-master/) **Links to preview builds**. Required for iOS, tvOS and watchOS support;
```

Comment:
Updating the version number here every time it's also updated in Make.config.inc seems redundant and tedious, maybe we can update the version number when the release is branched instead?